packages/forms-web-app/src/controllers/eligibility/granted-or-refused-permission.js

```diff
@@ -13,8 +20,8 @@ exports.postGrantedOrRefusedPermission = async (req, res) => {
   const { appeal } = req.session;
   const { errors = {}, errorSummary = [] } = body;
 
-  const planningPermissionStatus = body['granted-or-refused-permission'];
-
+  let planningPermissionStatus = body['granted-or-refused-permission'];
```

Comment:
The value of `planningPermissionStatus` isn't being changed so does it need to change from `const` to `let`?